### PR TITLE
remove redundant code from dict

### PR DESCRIPTION
**Pull request by mixmix**

### CODE REVIEW

File: dict.js [hunk 1]
```diff
@@ -129,12 +129,13 @@ function Dict (defaultValues, opts) {
 
   function listen () {
     Object.keys(sources).forEach(function (key) {
-      releases[key] = bind(sources[key])
+      releases[key] = null
```
Comment (mixmix) [inline dict.js]: your code will currently be equivalent to this (as you haven't passed bind a 2nd arg).
Is that inentional ?

File: dict.js [hunk 2]
```diff
@@ -129,12 +129,13 @@ function Dict (defaultValues, opts) {
...
     })
 
     if (opts && opts.onListen) {
       var release = opts.onListen()
       if (typeof release === 'function') {
+        // isn't releases an object? how are you pushing into it?
         releases.push(release)
```
Comment (mixmix) [inline dict.js]: this

**Comment by mmckegg** [comment]
whoops, you're right @mixmix 

Was a copy/paste mistake when I duped the bahavior from computed and then array. Found some other silly bugs in there too. Fixed in https://github.com/mmckegg/mutant/commit/066442e01f470f1b1ee4ac4e02c697ea0ac60721 and https://github.com/mmckegg/mutant/commit/150e4db298b51236bccc9dafe7866cc654fcf57b

**Comment by mixmix** [comment]
great, I thought I mighta been misreading. Thought a pull request would make what I was talking about clearer

**Pull request closed without merging**